tests/wallets/e2e/steps/helpers.js

```diff
@@ -139,8 +143,11 @@ export const getFixedAmountByName = async function(walletName: string) {
   await this.client.waitUntil(async () => {
     const isRestoring = await this.client.execute(
       (walletName) => {
-        const { isRestoring } = daedalus.stores.wallets.getWalletByName(walletName);
-        return isRestoring;
+        const wallet = daedalus.stores.wallets.getWalletByName(walletName);
+        if (wallet) {
+          return wallet.isRestoring;
+        }
+        return null;
```

Comment:
@yakovkaravelov this is not good. This will force test to fail because we are waiting until the wallet is in restoring state and with this check, you will say that this step is ok even if restoring is null. Rather throw an error if wallet not exists